content/docs/reference/buildpack-api.md

```diff
@@ -213,20 +230,55 @@ version = "10.x"
 
 [requires.metadata]
 version-source = ".nvmrc"
+```
 
-[[or]]
+As always, the buildpack `provides` `node`. In this particular case, a version of `node` (`10.x`) is being `requested` in a configuration file (`.nvmrc`), and the buildpack chooses to add an additional piece of metadata (`version-source`), so that it can understand where that request came from.
```

Comment:
It's a little odd to me to have `requested` in literal quotes since this doesn't appear to be something that's actually written out anywhere. But I get why you are emphasizing it.